### DISCUSSION TITLE
添加midway 邮箱组件用于发送邮件且支持模板渲染

### DISCUSSION THREAD

**Discussion by renjiali** [midwayjs/midway#3692] [Ideas]
### The feature, motivation（功能、动机）

添加midway 邮箱组件用于发送邮件且支持模板渲染

### Alternatives（替代方案）

wu

**Comment by renjiali** [comment]
@cyjake @we11adam @czy88840616 @ghostoy

**Reply by czy88840616** [reply]
我记得前两天有人提过一个来着https://midwayjs.org/docs/awesome_midway
